content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-probes.md

```diff
@@ -231,6 +237,46 @@ livenessProbe:
     port: liveness-port
 ```
 
+## Protect slow starting containers with startup probes
```

Comment:
(very minor suggestion, totally fine to merge without this)
```suggestion
## Protect slow starting containers with startup probes {#define-startup-probes}
```

A bit of future-proofing.  This lets the section title change separately to the anchor.